charts/spire/README.md

```diff
@@ -362,6 +362,12 @@ Now you can interact with the Spire agent socket from your own application. The
 | spire-server.ingress.tls | list | `[]` |  |
 | spire-server.initContainers | list | `[]` |  |
 | spire-server.jwtIssuer | string | `"oidc-discovery.example.org"` | The JWT issuer domain |
+| spire-server.keyManager.awsKMS.enabled | bool | `false` |  |
+| spire-server.keyManager.awsKMS.keyMetadataFile | string | `"/run/spire/data/aws-kms-key-metadata"` |  |
+| spire-server.keyManager.awsKMS.keyPolicyFile | string | `""` |  |
```

Comment:
Should the keyPolicyFile be a configmap reference instead so the user can upload it?

Reply:
Do you mean it can be just the name of a configmap instead of a file path? something like this

```yaml
awsKMS:
  keyPolicyConfigMap: my-policy
```
then a user can
```yaml
kind: ConfigMap
metadata:
  name: my-policy
data:
  policy.json: # policy here
```
then we mount the configmap, if set, on a volume and hard-code the `keyPolicyFile` in the config of the spire server?

That makes sense, but we'd need to establish a well-known file name (i.e `policy.json`) so we can safely mount it, right?

Reply:
Or maybe, something like:
```
awsKMS:
  keyPolicy:
    existingConfigMap: ""
    policy: ""
```

If they specify existingConfigMap, then include policy.json from it.
If policy, make a configmap for them with the policy content?